src/__tests__/ZBClient-unmocked.spec.ts

```diff
@@ -1,15 +1,17 @@
 import { ZBClient } from '..'
-jest.unmock('node-grpc-client')
 
 describe('ZBClient constructor', () => {
 	it('throws an exception when there is no broker and retry is false', async () => {
 		const zbc = new ZBClient('localhoster', { retry: false })
-		expect.assertions(1)
-		try {
-			await zbc.deployWorkflow('./test/hello-world.bpmn')
-		} catch (e) {
-			expect(e.message.indexOf('14 UNAVAILABLE:')).toEqual(0)
-		}
+		setTimeout(async () => {
```

Comment:
Is this setTimeout necessary? Would be good to keep these tests running fast.

Reply:
I had this test fail locally, until I added the timeout. So I figured its needed?!

Reply:
Hmmm... shouldn't be. I'll run the tests and see if I can figure it out.